examples/LaserWakefield/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -120,41 +128,51 @@ typedef Particles<ParticleDescription<
 struct Hydrogen
 {
     static const float_X numberOfProtons  = 1.0;
-    static const float_X numberOfNeutrons = 0.0;
+    static const float_X numberOfNeutrons = 1.0;
```

Comment:
Yeah sure but why not just add a `struct Deuterium`?
At some point we need a list of the most common elements anyway. 
@psychocoderHPC you can also create a new file for only the structs of these elements.